apps/dashboard/src/routes/org/[slug]/+layout.svelte

```diff
@@ -1,29 +1,37 @@
 <script>
   import { page } from '$app/stores';
+  import { goto } from '$app/navigation';
+  import { currentOrg } from '$lib/utils/store/org';
+
   import AddOrgModal from '$lib/components/Org/AddOrgModal/AddOrgModal.svelte';
   import { isQuizPage } from '$lib/utils/functions/app';
   import OrgSideBar from '$lib/components/Org/SideBar.svelte';
   import VerifyEmailModal from '$lib/components/Org/VerifyEmail/VerifyEmailModal.svelte';
   import Box from '$lib/components/Box/index.svelte';
-  import { currentOrg } from '$lib/utils/store/org';
-  import { goto } from '$app/navigation';
+  import PathwaySidebar from '$lib/components/Pathways/Sidebar.svelte';
 
   export let data;
 
   $: if ($currentOrg.id && data.orgName === '*') {
     const newUrl = $page.url.pathname.replace('*', $currentOrg.siteName);
     goto(newUrl + $page.url.search);
   }
+
+  $: containsPathways = $page.url?.pathname.includes('/pathways');
 </script>
 
 <AddOrgModal />
 
 <VerifyEmailModal />
 
 <div class="org-root w-full flex items-center justify-between">
-  {#if !isQuizPage($page.url?.pathname)}
+  <!-- this block switches the navigation from the existing one to the pathway's -->
+  {#if !isQuizPage($page.url?.pathname) && !containsPathways}
     <OrgSideBar />
   {/if}
+  {#if containsPathways}
```

Comment:
Please create a function like `isQuizPage` rather than creating a variable for this.

Something like `isPathwayPage`